products/terraform/docs/cloudngfw/azure/tutorials/intro.mdx

```diff
@@ -0,0 +1,246 @@
+  resource_group_name = azurerm_resource_group.rg.name
+  location            = var.region1
+  allocation_method   = "Static"
+  sku = "Standard"
+}
+
+resource "azurerm_public_ip" "cngfw-pip-westeu" {
+  name                = "cngfw-pip-westeu"
+  resource_group_name = azurerm_resource_group.rg.name
+  location            = var.region2
+  allocation_method   = "Static"
+  sku = "Standard"
+}
+```
+3. Virtual WAN
+
+Use the resources and data sources to interact with the vWAN-specific objects. For example,
+```
+resource "azurerm_virtual_wan" "vwan" {
+  name                = "terraform-vwan"
+  resource_group_name = azurerm_resource_group.rg.name
+  location            = var.region1
+}
+
+resource "azurerm_virtual_hub" "vhub-eastus" {
+  name                = "terraform-vhub-eastus"
+  resource_group_name = azurerm_resource_group.rg.name
+  location            = var.region1
+  virtual_wan_id      = azurerm_virtual_wan.vwan.id
+  address_prefix      = "10.0.0.0/23"
+}
+resource "azurerm_virtual_hub" "vhub-westeu" {
+  name                = "terraform-vhub-westeu"
+  resource_group_name = azurerm_resource_group.rg.name
+  location            = var.region2
+  virtual_wan_id      = azurerm_virtual_wan.vwan.id
+  address_prefix      = "10.0.0.0/23"
+}
+```
+Cloud NGFW resource requires an NVA object to be created in the target virtual hub:
+ ```
+resource "azurerm_palo_alto_virtual_network_appliance" "nva-eastus" {
+  name           = "terraform-nva-eastus"
+  virtual_hub_id = azurerm_virtual_hub.vhub-eastus.id
+}
+
+resource "azurerm_palo_alto_virtual_network_appliance" "nva-westeu" {
+  name           = "terraform-nva-westeu"
+  virtual_hub_id = azurerm_virtual_hub.vhub-westeu.id
+}
+```
+4. Local Rulestack
+
+```
+resource "azurerm_palo_alto_local_rulestack" "lrs-eastus" {
+  name                  = "terraform-lrs-eastus"
+  resource_group_name   = azurerm_resource_group.rg.name
+  location              = var.region1
+  anti_spyware_profile  = "BestPractice"
+  anti_virus_profile    = "BestPractice"
+  file_blocking_profile = "BestPractice"
+  vulnerability_profile = "BestPractice"
+  url_filtering_profile = "BestPractice"
+}
+```
+5. Cloud NGFW
+
+First, create a Cloud NGFW resource managed by Azure Rulestack
+
+```
+resource "azurerm_palo_alto_next_generation_firewall_virtual_hub_local_rulestack" "cngfw-eastus" {
+  name                = "terraform-cngfw-eastus"
+  resource_group_name = azurerm_resource_group.rg.name
+  rulestack_id        = azurerm_palo_alto_local_rulestack.lrs-eastus.id
+
+  network_profile {
+    public_ip_address_ids        = [azurerm_public_ip.cngfw-pip-eastus.id]
+    virtual_hub_id               = azurerm_virtual_hub.vhub-eastus.id
+    network_virtual_appliance_id = azurerm_palo_alto_virtual_network_appliance.nva-eastus.id
+  }
+}
+```
+Next, create a Cloud NGFW managed by Panorama. You are required to have a Panorama instance with the Azure plugin installed to generate the Registration String.
```

Comment:
Should we add "Panorama instance with the Azure plugin installed" as a pre-req in the begining of this article. 

Also if the registration string == panorama string probably a good idea use one attribute name throughout to avoid confusion

Reply:
Spelled out the attr